playbook/roles/filebeat/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Just remove the file / path if not needed

Reply:
ok